modules/Link.js

```diff
@@ -108,11 +119,10 @@ class Link extends React.Component {
         href={router ? router.createHref(to) : to}
         onClick={this.handleClick}
         style={isActive ? { ...style, ...activeStyle } : style }
-        className={isActive ?
-          [ className, activeClassName ].join(' ').trim() : className
-        }
+        className={className + (isActive ? " " + activeClassName : "")}
```

Comment:
for instance, when there is no `className` -> `<Link activeClassName="active"/>`